#Mode
uint8 mode

#Operation state
uint8 state

#Goal Position
float32 goalX
float32 goalY
float32 goalTheta

#Current position
float32 curX
float32 curY
float32 curTheta

#Goal velocity
float32 goalSpeed
float32 goalAngular

#Current velocity
float32 curSpeed
float32 curAngular
